plugin_description.xml

```diff
@@ -253,6 +253,11 @@
       Makes it easy to move around a given point on the XY plane, looking at it from any angle.
     </description>
   </class>
+  <class name="rviz/ThirdPersonFollower" type="rviz::ThirdPersonFollowerViewController" base_class_type="rviz::ViewController">
+    <description>
+      Makes it easy to move around a given point on the XY plane, looking at it from any angle.
```

Comment:
Please update the help text to reflect the fact that this controller also follow the orientation of the target frame.